DDuijker/Swiping-App
# Feature/ios build

### Pull Request Title: Add iOS Build Files and Update .gitignore

**Description:**
- Added the `ios` folder containing the native iOS build files required for running the application on iOS devices.
- Configured the project for iOS development builds, allowing team members to test features on the iOS simulator and physical devices.

**Changes Made:**
- Created the `ios` folder with necessary build configurations.
- Updated the `.gitignore` file to exclude unnecessary build files and folders that do not need to be tracked in Git, such as:
  - `ios/build/`
  - `ios/Pods/`
  - `ios/DerivedData/`

**How to Test:**
1. Clone the repository and pull the latest changes.
2. Ensure you have Xcode and the necessary development tools installed on your Mac.
3. Open the `ios` folder in Xcode and build the project.
4. Run the app on the iOS simulator or a connected device to verify that it works correctly.
